### DISCUSSION TITLE
Compilation on MacOS

### DISCUSSION THREAD

**Discussion by qoocku** [defun-games/claylib#76] [Q&A]
Hi all.
It happened that I work on MacBook so I'd need to get this work on `:darwin`. First, let me ask the following question:

- is `librayshim` needed for `claylib` at all?

Thnx in adv.

**Comment by shelvick** [comment]
Hey, sorry for the delay, I totally missed this.

`librayshim` is created as part of the claw wrappers to bypass some performance limitations in libffi. If the shim is causing problems it is definitely possible to use Claylib without it, but just know that you might hit performance bottlenecks a lot faster.

Unfortunately I don't have an up-to-date Mac to test on so I can't help much with actually getting it working there.
